src/app/recipes/recipes.component.ts

```diff
@@ -1,12 +1,21 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit} from '@angular/core';
+import {  Observable } from 'rxjs';
+import { Store, select } from '@ngrx/store';
+
+import * as fromApp from 'src/app/store/app.reducer';
+import * as RecipesSelectors from './store/recipe.selectors';
 
 @Component({
   selector: 'app-recipes',
   templateUrl: './recipes.component.html',
   styleUrls: ['./recipes.component.css'],
 })
-export class RecipesComponent implements OnInit {
-  constructor() {}
-
-  ngOnInit(): void {}
+export class RecipesComponent implements OnInit{
+  error: Observable<string> = null;
```

Comment:
you can immediately initialize this variable `error: Observable<string> = this.store.pipe(select(RecipesSelectors.error))`

Reply:
fixed